pages/content/posts/branding.md

```diff
@@ -0,0 +1,34 @@
+---
+author: "OWD team"
+title: "Announcing the launch of our new logo and website"
+date: "2022-10-03"
+tags: ["announcements"]
+ShowToc: false
+ShowBreadCrumbs: false
+excerpt: "Introducing a website redesign and a new logo, featuring Carle the bookworm"
+---
+
+Open Web Docs is pleased to announce the launch of a new website, a new logo, and Carle, the new Open Web Docs mascot.
+
+Steering committee member and [Black Girls Tech](https://www.blackgirlsintech.org/) founder, [Lola Odelola](https://lolaodelola.dev/), led the OWD rebranding efforts. She coordinated the development of the new OWD mascot and logo and built the new website. 
+
+Carle, pronounced *kahr-lee*, the OWD mascot, was designed by [Vladimir Grigoriev](https://nl.linkedin.com/in/grigza), a design lead at [JetBrains](https://Jetbrains.com). Carle is a bookworm! She’s named after the beloved children’s book author, [Eric Carle](https://eric-carle.com/). Carle personifies the OWD brand and is super memorable.  
+
+"I definitely love the new logo. It makes me happy every time I see it, and it perfectly reflects the marketing tone we started defining a year ago of fun, inclusion, accuracy, kindness, and goofiness. Hats off to Lola and Vladimir, and a huge thank you to JetBrains as well," remarked [Estelle Weyl](https://github.com/estelle), OWD staff writer. 
+
+While Carle is usually smiling, she has five other facial reactions, enabling using slightly modified logos with different moods for different purposes. Her main color is a fantastic purple orchid, #B448FF.
+
+<img width="400" alt="Carle the bookworm with six different facial expressions" src="https://user-images.githubusercontent.com/69888/192943190-9b938307-4084-49e5-86c6-0b270e09f6ad.png">
```

Comment:
Should we upload this image to a better place like `assets/img/`? Can you add it to this PR?